pkg/sdk/logging/model/syslogng/config/log.go

```diff
@@ -80,3 +99,20 @@ func destinationLogPath(dest destination) render.Renderer {
 		parenDefStmt("destination", render.Literal(dest.renderedDestName))),
 	)
 }
+
+func renderLabelGoTemplates(labels filter.ArrowMap, values any) filter.ArrowMap {
+	for k, v := range labels {
```

Comment:
```suggestion
	for k, v := range maps.Clone(m.Labels) {
```

this could come here, so we explicitly return a changed copy